backend/src/stwo/proof.rs

```diff
@@ -124,3 +126,14 @@ impl<B: Backend> From<StarkProvingKey<B>> for SerializableStarkProvingKey {
         Self { preprocessed }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+pub struct Proof<MC: MerkleChannel>
+where
+    MC::H: DeserializeOwned + Serialize,
+{
+    pub stark_proof: StarkProof<MC::H>,
+    pub constant_col_log_sizes: Vec<u32>,
+    pub witness_col_log_sizes: Vec<u32>,
+    pub machine_log_sizes: Vec<u32>,
```

Comment:
Maybe it's clearer to have `BTreeMap`s here instead of vectors, indexed by machine name and column name?

Reply:
Can be another PR